Packs/Syslog/Integrations/SyslogSender/SyslogSender.yml

```
@@ -11,7 +11,7 @@ configuration:
   required: false
   type: 0
   section: Connect
-  additionalinfo: A Syslog server address must be provided.
+  additionalinfo: A Syslog server address must be provided. Note, max supported length of label inside an address is 64 characters.
```

```suggestion
  additionalinfo: A Syslog server address must be provided. Note: the maximal supported length of a label inside an address is 64 characters.
```
not sure if the term "label" is clear for everyone, is there an alternative term?